typescript/index.js

```diff
@@ -5,7 +5,17 @@ module.exports = {
   ],
   rules: {
     '@typescript-eslint/indent': ['error', 2],
-    '@typescript-eslint/camelcase': false,
+    '@typescript-eslint/camelcase': 'off',
+    'import/extensions': [
+      'error',
+      'ignorePackages',
+      {
+        'js': 'never',
```

Comment:
I can't find it now but I remember reading the spec requires extensions for import. Any reason for disallowing extensions? What about ignoring the rule instead for now?